tests/test_redis_communication.py

```diff
@@ -253,11 +253,12 @@ def test_event_listener_calls_the_event_listener_of_the_root_dispatcher(self, ma
             dispatcher.redis.publish.assert_called_once()
 
     def test_publish_event_converts_python_objects_to_json(self):
-        offer = Offer("1", now(), 2, 3, "A")
-        trade = Trade("2", now(), Offer("accepted", now(), 7, 8, "Z"), "B", "C",
+        now = datetime(2021, 11, 3, 10, 45)
+        offer = Offer("1", now, 2, 3, "A")
+        trade = Trade("2", now.add(minutes=1), Offer("accepted", now, 7, 8, "Z"), "B", "C",
                       None, None, TradeBidOfferInfo(None, None, None, None, None))
-        new_offer = Offer("3", now(), 4, 5, "D")
-        existing_offer = Offer("4", now(), 5, 6, "E")
+        new_offer = Offer("3", now, 4, 5, "D")
+        existing_offer = Offer("4", now, 5, 6, "E")
```

Comment:
Why did we remove the now function calls?

Reply:
In favor of `now = datetime(2021, 11, 3, 10, 45)` in line 256